README.md

```diff
@@ -1,5 +1,3 @@
-# discussions
+# [ARCHIVED] Hackathon Port of Discussions to Python
```

Comment:
I have also put in an IT help request to make this the description of the repo. While it isn't necessary for the README to start with the same thing, I thought it was more useful than "discussions".